.env.example

```
@@ -12,4 +12,7 @@
 DATABASE_URL=postgresql://root@localhost:26259/misskey_list?sslmode=disable
 
 # Task API authentication secret (for Cloud Scheduler)
-TASK_SECRET=your-secret-key-here
\ No newline at end of file
+TASK_SECRET=your-secret-key-here
+
+ADMIN_USER=admin
+ADMIN_PASSWORD=admin
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ADMIN_PASSWORD key should go before the ADMIN_USER key

(UnorderedKey)